src/locales.json

```diff
@@ -77,6 +77,16 @@
       "lastUpdate": "آخرین به روزآوری: "
     }
   },
+  "common": {
+    "en": {
+      "notAvailable": "Not Available",
```

Comment:
This key is already present in the locales.json as `NotFoundLanguage`: Sorry, but your language is not available here. Also see `ChooseALanguage`

https://docs.google.com/spreadsheets/d/1kMxVlqzwjjDoARO5Ss5oMNUEjqubT0Ps8NB7cguP-yU/edit#gid=194836756

Reply:
Right, I should have checked that. I replaced the keys.